### PR TITLE
add Staking modules section, ET section small restructure

### PR DESCRIPTION
**Pull request by zuzueeka**
add Staking modules section, ET section small restructure

### CODE REVIEW

File: docs/deployed-contracts/holesky.md [hunk 1]
```diff
@@ -10,9 +10,7 @@ We're excited to announce that the deployment campaign for Lido on Holešky is c
 Please be aware that you might encounter unexpected behavior during this period. We're working diligently to ensure a smooth deployment, but unforeseen issues can arise.
 :::
 
-## Holešky testnet
-
-### Lido V2 Core Protocol
+## Lido V2 Core Protocol
```
Comment (TheDZhon) [inline docs/deployed-contracts/holesky.md:13]: Maybe we can omit 'Lido V2' prefixes here and there (I know it affects not only Holešky :smile: )

File: docs/deployed-contracts/holesky.md [hunk 2]
```diff
@@ -65,12 +63,25 @@ Please be aware that you might encounter unexpected behavior during this period.
 - Gate Seal Factory: [`0x1134F7077055b0B3559BE52AfeF9aA22A0E1eEC2`](https://holesky.etherscan.io/address/0x1134F7077055b0B3559BE52AfeF9aA22A0E1eEC2)
 - Gate Seal: [`0x7f6FA688d4C12a2d51936680b241f3B0F0F9ca60`](https://holesky.etherscan.io/address/0x7f6FA688d4C12a2d51936680b241f3B0F0F9ca60)
 
-### Easy Track
+## Staking modules
+
+- Curated Node Operators: [`0x595F64Ddc3856a3b5Ff4f4CC1d1fb4B46cFd2bAC`](https://holesky.etherscan.io/address/0x595F64Ddc3856a3b5Ff4f4CC1d1fb4B46cFd2bAC)
+- Sandbox: [`0xD6C2ce3BB8bea2832496Ac8b5144819719f343AC`](https://holesky.etherscan.io/address/0xD6C2ce3BB8bea2832496Ac8b5144819719f343AC)
```
Comment (TheDZhon) [inline docs/deployed-contracts/holesky.md:69]: Maybe we should explain what 'Sandbox' means (it hasn't been announced anywhere, I mean)

File: docs/deployed-contracts/holesky.md [hunk 3]
```diff
@@ -65,12 +63,25 @@ Please be aware that you might encounter unexpected behavior during this period.
...
+- Simple DVT: [`0x11a93807078f8BB880c1BD0ee4C387537de4b4b6`](https://holesky.etherscan.io/address/0x11a93807078f8BB880c1BD0ee4C387537de4b4b6)
```
Comment (TheDZhon) [inline docs/deployed-contracts/holesky.md:70]: I would rearrange 'Sandbox' and 'Simple DVT'

**Review by TheDZhon — APPROVED** [review]
LGTM, however I suggest some additional improvements if possible :pray:

File: docs/deployed-contracts/holesky.md (hunk 1 above)
Comment (zuzueeka) [reply docs/deployed-contracts/holesky.md:13]: I would remove this synchronously with the mainnet page to maintain consistency

File: docs/deployed-contracts/holesky.md (hunk 3 above)
Comment (zuzueeka) [reply docs/deployed-contracts/holesky.md:69]: yes, it’s more logical. changed it

File: docs/deployed-contracts/holesky.md (hunk 2 above)
Comment (zuzueeka) [reply docs/deployed-contracts/holesky.md]: this is a module for testing, onboarding contributors and all that stuff to try. I think sandbox is quite good naming

**Review by TheDZhon — APPROVED** [review]

**Comment by TheDZhon** [comment]
Please consider integrating #296

**Review by grstepanov — APPROVED** [review]
LGTM!

**Pull request merged**